packages/constants/src/lib/constants/constants.ts

```diff
@@ -688,32 +683,31 @@ export const metamaskChainInfo = {
    * Information about the "chronicleVesuvius" chain.
    */
   chronicleVesuvius: {
-    chainId: LIT_CHAINS['chronicleVesuviusTestnet'].chainId,
-    chainName: LIT_CHAINS['chronicleVesuviusTestnet'].name,
+    chainId: LIT_CHAINS['vesuvius'].chainId,
+    chainName: LIT_CHAINS['vesuvius'].name,
     nativeCurrency: {
-      name: LIT_CHAINS['chronicleVesuviusTestnet'].symbol,
-      symbol: LIT_CHAINS['chronicleVesuviusTestnet'].symbol,
-      decimals: LIT_CHAINS['chronicleVesuviusTestnet'].decimals,
+      name: LIT_CHAINS['vesuvius'].symbol,
+      symbol: LIT_CHAINS['vesuvius'].symbol,
+      decimals: LIT_CHAINS['vesuvius'].decimals,
     },
-    rpcUrls: LIT_CHAINS['chronicleVesuviusTestnet'].rpcUrls,
-    blockExplorerUrls: LIT_CHAINS['chronicleVesuviusTestnet'].blockExplorerUrls,
+    rpcUrls: LIT_CHAINS['vesuvius'].rpcUrls,
+    blockExplorerUrls: LIT_CHAINS['vesuvius'].blockExplorerUrls,
     iconUrls: ['future'],
   },
 
   /**
    * Information about the "chronicleYellowstone" chain.
    */
   chronicleYellowstone: {
```

Comment:
As above, since this was never merged into the live/shipped SDK, should we just :hocho: it and only ship `yellowstone`?